tst/EnergyPlus/unit/HeatBalanceKivaManager.unit.cc

```diff
@@ -212,7 +212,7 @@ TEST_F(EnergyPlusFixture, HeatBalanceKiva_SetInitialBCs)
 
     // Test using default Initial Indoor Temperature with Cooling/Heating Setpoints of 24C/20C
 
-    Real64 coolingSetpoint2 = 24.0;
+    Real64 heatingSetpoint2 = 20.0;
```

Comment:
Tests were updated because they use [an outdoor temperature of 10 degC](https://github.com/NREL/EnergyPlus/blob/6b63fff511cd04ed50965993c7eb8826b1a9c867/tst/EnergyPlus/unit/HeatBalanceKivaManager.unit.cc#L117), which [corresponds to the heating setpoint, not the cooling setpoint](https://github.com/NREL/EnergyPlus/blob/6b63fff511cd04ed50965993c7eb8826b1a9c867/src/EnergyPlus/HeatBalanceKivaManager.cc#L302).